src/mongo/registerMongoCommands.ts

```diff
@@ -148,3 +163,23 @@ function setConnectedNode(node: IAzureNode | undefined, codeLensProvider: MongoC
     let dbName = node && node.treeItem.label;
     codeLensProvider.setConnectedDatabase(dbName);
 }
+
+function updateErrorsInActiveDocument(context: vscode.ExtensionContext): void {
+    callWithUnxpectedErrorTelemetry(
```

Comment:
See my comments in this PR: https://github.com/Microsoft/vscode-azuretools/pull/189, but I'm not sure I see the value in callWithUnxpectedErrorTelemetry.

I would prefer to follow the pattern of  [other events](https://github.com/Microsoft/vscode-cosmosdb/blob/master/src/extension.ts#L117), which use `actionHandler.registerEvent` and `this.suppressTelemetry`